src/drivers/mct/main/prep_rof_mod.F90

```diff
@@ -489,6 +492,7 @@ subroutine prep_rof_merge(l2x_r, a2x_r, fractions_r, x2r_r, cime_model)
        index_x2r_Flrl_rofsub = mct_aVect_indexRA(x2r_r,'Flrl_rofsub' )
        index_x2r_Flrl_rofdto = mct_aVect_indexRA(x2r_r,'Flrl_rofdto' )
        index_x2r_Flrl_rofi   = mct_aVect_indexRA(x2r_r,'Flrl_rofi' )
+       index_x2r_Flrl_demand = mct_aVect_indexRA(x2r_r,'Flrl_demand' )
```

Comment:
What is the reason for this addition? It looks like `index_x2r_Flrl_demand` is already set in a conditional on e3sm a few lines up, and it looks like it's only needed if running with e3sm.

Reply:
I have removed the duplicate line.